### DISCUSSION TITLE
Nuevos tipos de selectores para CMS

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1257] [Proposals]
### Goal

Conseguir reducir el nº de elementos que usan MUI y que se usen inputs de Mistica.

### Context

A día de hoy usamos varios componente de MUI porque los de Mística no cubren las necesidades de producto. Necesitaríamos desarrollar esos inputs en Mistica.

### Impact

Hasta el momento los inputs que hemos detectado son:
-Selector que permita incluir iconos o imágenes en la lista de opciones
<img width="678" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/c3d92c31-8cca-40ac-9f07-11468f32dd9b">

-Selector con autocomplete:
https://semantic-ui.com/modules/dropdown.html#search-selection

-Selector con autocomplete y tags
https://semantic-ui.com/modules/dropdown.html#multiple-selection

### Discussion Questions

¿Sería posible que Mistica incluya esos casos o debemos seguir usando Mui?

**Comment by yceballost** [comment]
Estamos de acuerdo en que es un componente que debe existir en Mística, es un componente complejo y no tenemos priorización sobre cuándo hacer esto. Si lo quieres mover para Q2 u otro Q por nosotros perfecto y si no de momento usad MUI
